### ISSUE TITLE
Test and validate VBN monitor_delay implementation

### ISSUE THREAD

**Issue by wbwakeman** [AllenInstitute/ecephys_etl_pipelines#20]
Follow up to #18, need to process test data and get people to review the outputs to confirm

Tasks:
* Create validation data files
* Process them with the new code
* Provide outputs to key stakeholders (e.g. Nick, Corbett) for review
* Address feedback

**Comment by danielsf** [comment]
I'd like to put in a specific call for unit tests to these methods
`trim_border_pulses`
`correct_on_off_effects`
`fix_unexpected_edges`
and
`flag_unexpected_edges`

They seem very self-contained yet have interesting edge cases that we need to be handling correctly.

**Comment by ns706** [comment]
I have run multiple validation files with the new code and have provided the outputs to Corbett. A few issues came up (see issues #25 and #24) which I have been addressed. 

I've also fixed many of the broken tests. However, I could not fix all of them as a few are much more involved and cannot be easily fixed. A smaller version of a real experiment will probably be needed as a test fixture for these cases.
